TopMovies/Resources/Routing/RouterTransition.swift

```diff
@@ -8,4 +8,5 @@
 enum RouteType {
   case allCategories
   case category(_ categoryId: MovieCategory.ID)
+  case movie(_ movieId: MoviePreview.ID)
```

Comment:
What's the difference between
```swift
case category(_ categoryId: MovieCategory.ID)
case movie(_ movieId: MoviePreview.ID)
```
and
```swift
case category(MovieCategory.ID)
case movie(MoviePreview.ID)
```
? :)

Reply:
:)